config/build/webpack.conf.js

```diff
@@ -56,13 +57,13 @@ module.exports = (mode) => {
 	 */
 	if (test) {
 		wpConfig.entry = {
-			application: path.posix.resolve('./src/client/main.ts')
+			application: path.posix.resolve(clientPath+'/main.ts')
```

Comment:
Each of these should be something more like:
path.posix.join(clientPath, 'main.ts')

path.posix.resolve() converts a path to an absolute path. So really, we might want to make clientPath = './src/client' and then do the above.

path.posix.join will combine paths (without having to worry about leading/trailing slash wonkiness), which is why I recommend the change